formfactory/admin.py

```diff
@@ -63,13 +62,30 @@ class WizardActionThroughInline(admin.StackedInline):
 class WizardAdmin(admin.ModelAdmin):
     form = forms.WizardAdminForm
     model = models.Wizard
-    list_display = ["title"]
-    inlines = [FormThroughInline, WizardActionThroughInline]
+    inlines = [WizardFormThroughInline, WizardActionThroughInline]
     prepopulated_fields = {"slug": ["title"]}
 
 
+class FieldGroupThroughInline(admin.StackedInline):
+    form = forms.FieldGroupThroughAdminForm
+    model = models.FieldGroupThrough
+
+
+class FormFieldGroupAdmin(admin.ModelAdmin):
```

Comment:
ModelAdmin is inherited by everything and is django native, so would require a monkey patch which would affect all classes inheriting from ModelAdmin. A BaseModelAdmin class could be added but it's not necessary for such a small set of admin classes which require the prepopulate fields var.